tools/NuGet/PostArtifactoryPackages.bat

```diff
@@ -0,0 +1,9 @@
+:: Argument %1: API key to access the Artifactory server
+::
+
+call .\BuildPackages.bat "template-artifactory"
+
+set Artifactory=https://art-bobcat.autodesk.com/artifactory/api/nuget/team-dynamo-nuget
+
+:: Push Runtime package to Artifactory
+nuget push *.nupkg -source %Artifactory% -apikey %1
```

Comment:
:: Use credentials in config file for pushing package to Artifactory
SET ConfigPath=%~dp0..\..\dynamo-nuget.config
nuget push DynamoVisualProgramming.DynamoCoreRuntime.*.nupkg -source %Artifactory% -configfile %ConfigPath%